openapi-contracts/src/test/resources/domains/verification/v1/start/request/VerificationStartRequestSmsDto.json

```
@@ -9,6 +9,7 @@
   "smsOptions": {
     "expiry": "01:02:03",
     "codeType": "Alphanumeric",
-    "template": "My template require to use '{{CODE}}' code"
+    "template": "My template require to use '{{CODE}}' code",
```

Nit: This is very unlikely a real-life message but at least let's correct the error
```suggestion
    "template": "My template requires to use {{CODE}} placeholder",
```

---

Done